### ISSUE TITLE
Say It All: Feedback for Improving Non-Visual Presentation Accessibility

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#403]
### Links
- Paper: https://dl.acm.org/doi/10.1145/3411764.3445572

### Abstract
Presenters commonly use slides as visual aids for informative talks. When presenters fail to verbally describe the content on their slides, blind and visually impaired audience members lose access to necessary content, making the presentation difficult to follow. Our analysis of 90 presentation videos revealed that 72% of 610 visual elements (e.g., images, text) were insufficiently described. To help presenters create accessible presentations, we introduce Presentation A11y, a system that provides real-time and post-presentation accessibility feedback. Our system analyzes visual elements on the slide and the transcript of the verbal presentation to provide element-level feedback on what visual content needs to be further described or even removed. Presenters using our system with their own slide-based presentations described more of the content on their slides, and identified  3.26 times more accessibility problems to fix after the talk than when using a traditional slide-based presentation interface. Integrating accessibility feedback into content creation tools will improve the accessibility of informational content for all. 

<img width="867" alt="ScreenShot 2021-07-03 1 16 56" src="https://user-images.githubusercontent.com/26996041/124302639-6de46d80-db9c-11eb-9fa6-7718ae895b61.png">
<img width="880" alt="ScreenShot 2021-07-03 1 17 04" src="https://user-images.githubusercontent.com/26996041/124302641-6e7d0400-db9c-11eb-82ed-182592346c3c.png">
<img width="883" alt="ScreenShot 2021-07-03 1 17 11" src="https://user-images.githubusercontent.com/26996041/124302643-6f159a80-db9c-11eb-8ee6-e2360f5edeaf.png">

- 視覚障害者の聴衆がいたときに，スライド中のどこを追加で説明するべきか，削除すべきかを提示してくれるシステム
